packages/amplify_datastore/ios/internal/AWSDataStorePlugin/AWSDataStorePlugin+DataStoreBaseBehavior.swift

```diff
@@ -0,0 +1,601 @@
+//
```

Comment:
*suggestion:* Can we update `.gitattributes` to treat everything under /internal as generated code, similar to what we do for generated pigeon code? This will make it so it is collapsed by default in code review. It won't be something we typically want to review in detail in code reviews.

It would also be nice if we could add something like this to the top of each file, but this is just a nice to have `// generated code. do not edit. - run 'aft ...' to pull in latest from amplify-swift`

Reply:
Good suggestion, I added the directory to `.gitattributes`. 

I also agree it would be nice to have that header call out. After a quick glance, it appears that adding data at the beginning of a file is more involved. Which seems to add more complexity than its worth for this use case.